css/properties.json

```diff
@@ -2261,6 +2261,78 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border"
   },
+  "border-block": {
+    "syntax": "<'border-width'> || <'border-style'> || <'color'>",
```

Comment:
according to the spec, should be `<'border-top-width'> || <'border-top-style'> || <'color'>` ?

And same pattern for the other properties that have the same syntax?